docs/developer/order-expiration.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Order Expiration
+---
+
+:::info
+
+expireOrdersAfter was introduced in **Saleor 3.13**
+
+deleteExpiredOrdersAfter was introduced in **Saleor 3.14**
+
+:::
+
+:::caution
+
+This feature is in the **Feature Preview** stage, which means that it is available for experimentation and
+feedback. However, it is still undergoing development and is subject to modifications.
+
+:::
+
+## Introduction
+
+Saleor allows to finalize transactions without payment (order first, then payment). If the customer cancel payment process and order remains unpaid for some time, there is possibility to set automatic stock releases and remove order.
+
+## Configuration
+
+To set the expiration of orders, turn on the `automaticallyConfirmAllNewOrders` option in the channel settings and specify the number of minutes after which the orders should expire by setting `expireOrdersAfter`.
+Additionally, you can modify `deleteExpiredOrdersAfter`. By default, this is set to 60 days, but you can change this value to anywhere between 1 and 120 days.
+
+```graphql
+mutation ChannelUpdate {
+  channelUpdate(
+    id: "Q2hhbm5lbDoyMjQz"
+    input: {
+      orderSettings: {
+        expireOrdersAfter: 360
+        deleteExpiredOrdersAfter: 30
+        automaticallyConfirmAllNewOrders: true
+      }
+    }
+  ) {
+    channel {
+      id
+      orderSettings {
+        expireOrdersAfter
+        deleteExpiredOrdersAfter
+        automaticallyConfirmAllNewOrders
+      }
+    }
+    errors {
+      field
+      code
+      message
+    }
+  }
+}
+```
+
+```json
+{
+  "data": {
+    "channelUpdate": {
+      "channel": {
+        "id": "Q2hhbm5lbDoyMjQz",
+        "orderSettings": {
+          "expireOrdersAfter": 360,
+          "deleteExpiredOrdersAfter": 30,
+          "automaticallyConfirmAllNewOrders": true
+        }
+      },
+      "errors": []
+    }
+  },
+  "extensions": {
+    "cost": {
+      "requestedQueryCost": 0,
+      "maximumAvailable": 50000
+    }
+  }
+}
+```
+
+## How it works
+
+Once an unpaid order is created (via [`checkoutCreate`](api-reference/checkout/mutations/checkout-create.mdx) or [`orderCreateFromCheckout`](api-reference/orders/mutations/order-create-from-checkout.mdx)), it remains in `UNCONFIRMED` status. If the customer does not complete the payment within `expireOrdersAfter` minutes after that, the status will change to `EXPIRED` and stocks allocation will be released.
+After `deleteExpiredOrdersAfter` days orders in `EXPIRED` status will be deleted and disappear from the order list.
+
+## How to disable order expiration
+
+To disable order expiration, set `expireOrdersAfter` to 0.
+
+When `expireOrdersAfter` is set to 0, the order will not expire and will not be deleted.
```

Comment:
> and will not be deleted.

I think this could be misleading, you could provide here more context, like explaining, that when `expireOrdersAfter` is turned off, the logic responsible for deleting expired orders won't make anything.